.github/workflows/release.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       - uses: oxidize-rb/actions/setup-ruby-and-rust@v1
         with:
           ruby-version: "3.3"
-          bundler-cache: false
+          bundler-cache: true
```

Comment:
Also unrelated, but that sped up things and I can't see a reason not to use it. It's set to true elsewhere.